maintainers/maintainer-list.nix

```diff
@@ -8297,6 +8297,12 @@
     githubId = 474343;
     name = "Xavier Zwirtz";
   };
+  savannidgerinel = {
+    email = "savanni@luminescent-dreams.com";
+    github = "savannidgerinel";
+    githubId = 8534888;
+    name = "Savanni D'Gerinel";
+  };
```

Comment:
I believe this has to be alphabetical?

Reply:
Done. That wasn't entirely clear, but it looks like the end of the file has just gotten out of order.